.vscode/settings.json

```diff
@@ -0,0 +1,5 @@
+{
+    "python.linting.flake8Enabled": true,
+    "python.linting.enabled": true,
+    "python.languageServer": "Pylance",
+}

```

Comment:
Here are some additional settings for Python / linting type stuff:

```json
{
    "editor.formatOnSave": true,
    "files.encoding": "utf8",
    "files.eol": "\n",
    "files.insertFinalNewline": true,
    "files.trimFinalNewlines": true,
    "files.trimTrailingWhitespace": true,
    "python.formatting.provider": "black",
}
```

We'll want to add `black` and `flake8` to the [`pip install` in the Dockerfile](https://github.com/cal-itp/eligibility-api/blob/main/.devcontainer/Dockerfile#L18) though